src/js/pages/catalog/PackagesTab.js

```diff
@@ -208,16 +213,27 @@ class PackagesTab extends mixin(StoreMixin) {
       </p>
     );
     let title = "Community";
-    const isSearchActive = this.state.searchString !== "";
     const titleClasses = classNames("flush-top", {
       short: !isSearchActive,
       tall: isSearchActive
     });
 
     if (isSearchActive) {
       const foundPackagesLength = packages.getItems().length;
-      const packagesWord = StringUtil.pluralize("service", foundPackagesLength);
+      if (foundPackagesLength < 1) {
+        const noResults = `No results were found for your search: "${
```

Comment:
I personally see no benefit in this being not inlined, I am just curious why you decided to pull this out in a variable :)

Reply:
That was an existing variable - it is used as well [here](https://github.com/dcos/dcos-ui/blob/00d40ed81619b90ddbaaae9b6abadec5bb91a78a/src/js/pages/catalog/PackagesTab.js#L236). Should I inline in both places?

Reply:
no, I was just wondering :D